cogs/mod.py

```diff
@@ -170,109 +170,38 @@ def check_if_target_is_staff(self, target):
         userlog(target.id, ctx.author, reason, "bans", target.name)
 
         safe_name = await commands.clean_content().convert(ctx, str(target))
-
-        dm_message = f"You were banned from {ctx.guild.name}."
-        if reason:
-            dm_message += f" The given reason is: \"{reason}\"."
-        dm_message += "\n\nThis ban does not expire."
-
-        try:
-            await target.send(dm_message)
-        except discord.errors.Forbidden:
-            # Prevents ban issues in cases where user blocked bot
-            # or has DMs disabled
-            pass
+        
+        ban_type = "Ban"
+        if ctx.invoked_with in ["softban","silentban"]:
+            ban_type = "Silent Ban"
+            dm_message = f"You were banned from {ctx.guild.name}."
+            if reason:
+                dm_message += f" The given reason is: \"{reason}\"."
+            dm_message += "\n\nThis ban does not expire."
+
+            try:
+                await target.send(dm_message)
+            except discord.errors.Forbidden:
+                # Prevents ban issues in cases where user blocked bot
+                # or has DMs disabled
+                pass
```

Comment:
Instead of doing all this, why don't we simply do an if here, check if `invoked_with` is **not** silentban and only DM them if it is not?

Reply:
Haha... Not sure how I let that pass.